libr/bin/p/bin_java.c

```diff
@@ -102,7 +103,7 @@ static bool load(RBinFile *bf) {
 		return false;
 	}
 
-	bin_obj = load_bytes (bf, bytes, sz, bf->o->loadaddr, bf->sdb);
+	load_bytes (bf, (void **) &bin_obj, bytes, sz, bf->o->loadaddr, bf->sdb);
```

Comment:
is the cast really needed?

Reply:
I though it was needed, because the code (that I kept as close as possible to the original), uses `struct r_bin_java_obj_t *bin_obj = NULL` and `load_bytes()` expects a RBinFile*.

Reply:
Oh i see.